RevenueCat/Scripts/Purchases.cs

```diff
@@ -720,6 +933,17 @@ public void CollectDeviceIdentifiers()
     {
         _wrapper.CollectDeviceIdentifiers();
     }
+    
+    /// <summary>
+    /// Callback function containing the result of CanMakePayments
+    /// <param name="canMakePayments">A bool value indicating whether billing
+    /// is supported for the current user (meaning IN-APP purchases are supported),
+    /// and, if provided, whether a list of specified BillingFeatures are supported.
+    /// This will be false if there is an error</param>
+    /// <param name="error">An Error object or null if successful.</param>
```

Comment:
I've been wondering why sometimes we are adding the `<param>` inside the `<summary>` tag and sometimes outside. Does it organize the docs differently?

Reply:
it should always be outside

Reply:
did a pass through this file moving all params out of summary, thanks for calling this out!!